packages/perseus/src/widgets/__tests__/group_test.jsx

```diff
@@ -21,6 +21,12 @@ describe("group widget", () => {
         jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
             testDependencies,
         );
+        jest.useFakeTimers("modern");
+        jest.setSystemTime(Date.parse("04 Dec 1995 00:12:00 GMT"));
+    });
+
+    afterEach(() => {
+        jest.useRealTimers();
```

Comment:
Are we sure we need these in all these tests? And if so, why do we need them now when we didn't need them before?

Reply:
In webapp, I recently changed the default to be `useRealTimers()`.  We should probably consider doing the same for perseus as well.  It makes the tests run a bit slower, but I think a fair amount of tests require it anyways and we'll need to switch whenever we upgrade `@testing-library/user-events`.

Reply:
I don't think we need to change the default as part of this PR.  I can do that in a separate PR.

Reply:
This is part of the workaround I developed for creating unique IDs. Initially I wanted to use useUniqueId, but we ran into some errors that we couldn't debug. The workaround creates IDs based on DateTime. This caused issues for snapshot testing, so using the FakeTimer allows us to bypass that issue. I will add this explanation to the code as a comment and point it out in the PR summary as well.

Reply:
I'll try to dig into the issue with `useUniqueId*` sometime this week.

Reply:
Thanks, Kevin!

Also, I fixed the tests so setSystemTime is added only to tests that need it. I looks like tests by default use Fake timers, so I just removed all the references to setting real or fake timers. If we change the default to be real timers, I can go back and add use fake timers to the tests that need it.